app/view/signup/_signup.scss

```diff
@@ -0,0 +1,130 @@
+@import "_theme.scss";
+
+p {
+  text-align: center;
+}
+
+form {
+  h3 {
+    margin: 0px auto 10px auto;
+  }
+}
+
+.about {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  flex-wrap: wrap;
+  background-color: #383838;
+  width: 100%;
+  height: auto;
+  margin-top: 2px;
+
+  .about-gist, .about-contd {
+    text-align: center;
+    height: 175px;
+    outline: 2px solid white;
+
+    h3 {
+      font-size: 2em;
+      font-weight: normal;
+      padding-top: 10px;
+      margin-top: 0px;
+      margin-left: 10px;
+      margin-bottom: 0px;
+      color: $app-light-purple;
+    }
+
+    p {
+      text-align: justify;
+      color: white;
+      width: 300px;
+      margin-left: auto;
+      margin-right: auto;
+    }
+
+  }
+
+}
+
+.about-splash {
+  height: 300px;
+  width: auto;
+  background-image: url("ramble-about-splash.jpg");
+}
+
+.about-splash {
+
+  div {
+    padding: 1px;
+    max-width: 400px;
+    height: 100%;
+    margin: auto;
+  }
+
+  h1{
+    font-weight: normal;
+    font-family: sans-serif;
+    font-size: 7em;
+    color: white;
+    margin-top: 15%;
+    margin-bottom: 0px;
+    text-shadow: 2px 2px $app-light-purple;
+
+  }
+
+  h2{
+    font-weight: normal;
+    font-family: sans-serif;
+    font-size: 3em;
+    color: white;
+    margin: 0px;
+    margin-left: 175px;
+    text-shadow: 2px 2px $app-light-purple;
+  }
+
+}
+
+// iphone 6
+@media only screen
+  and (min-device-width: 375px)
+  and (max-device-width: 667px)
+  and (-webkit-min-device-pixel-ratio: 2)
+  and (orientation: portrait) {
+
+    body {
+
+      .about {
+        width: 100%;
+        flex-direction: column;
+      }
+      .about-gist, .about-contd {
+        display: none;
+      }
+    }
+}
+
+//iphone 5
+@media only screen
```

Comment:
great use of media queryies through out this app :)

Im super impressed that yall choose not to use bootstrap, and go a head and write all your styles from scratch using sass !